test/fixtures/long-running/jobs.yml.erb

```diff
@@ -1,3 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: multi-replica
+spec:
+  ports:
+  - port: 80
+    name: http
```

Comment:
It's because I'm shoehorning that existing test into also being able to assert proper logging for a service whose number of endpoints changes. The test is doing exactly what I'd want to do for this new test (see https://github.com/Shopify/kubernetes-deploy/pull/116/files#r125535926) and it inherently is longer-running than most. My thinking was that the ideological mismatch of adding a service to the template was a lesser evil than adding an extra 15s to every test run just for a logging assertion. Is that terrible?